package.json

```diff
@@ -17,25 +17,28 @@
   "license": "ISC",
   "peerDependencies": {
     "eslint": "6.2.0",
+    "eslint-plugin-detect-bad-words": "1.0.7",
     "eslint-plugin-filenames": "1.2.0",
     "eslint-plugin-folders": "1.0.3",
-    "eslint-plugin-react": "7.31.10",
-    "eslint-plugin-detect-bad-words": "1.0.7",
     "eslint-plugin-mocha": "10.1.0",
     "eslint-plugin-no-else": "0.2.2",
-    "eslint-plugin-sonarjs": "0.15.0",
-    "eslint-plugin-prefer-arrow": "1.2.3"
+    "eslint-plugin-prefer-arrow": "1.2.3",
+    "eslint-plugin-react": "7.31.10",
+    "eslint-plugin-sonarjs": "0.15.0"
   },
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.54.0",
+    "@typescript-eslint/parser": "^5.54.0",
```

Comment:
Use exact versions (unless you build a library). 

   "@typescript-eslint/eslint-plugin": "5.54.0",